templates/blocks/header.twig

```diff
@@ -0,0 +1,22 @@
+<div class="page-header col-md-8 col-lg-8 col-sm-12">
+    <div class="top-tags-wrap">
+      {% if page_tags %}
+        {% for page_tag in page_tags %}
+          <span>#{{ page_tag.name }}</span>
```

Comment:
To escape output in twig that is withing an html tag:
{{ page_tag.name|e('html') }}

Reply:
I've recommended a different approach for this: #3.

Reply:
Ok understood